From 40bddabd9a3258cb31e079e25f9723f354e8b2bb Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 23 Sep 2014 14:56:10 -0400
Subject: [drm] bochs: replace ALIGN(PAGE_SIZE) by PAGE_ALIGN

Message-id: <1411484172-28849-8-git-send-email-kraxel@redhat.com>
Patchwork-id: 96489
O-Subject: [RHEL-7.1 kernel PATCH 7/9] drm/bochs: replace ALIGN(PAGE_SIZE) by PAGE_ALIGN
Bugzilla: 1043379
RH-Acked-by: Dave Airlie <airlied@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

use mm.h definition

Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: David Herrmann <dh.herrmann@gmail.com>
(cherry picked from commit a8ba29cd337cce06899e1ad51c69b4beea7628a0)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/bochs/bochs_mm.c b/drivers/gpu/drm/bochs/bochs_mm.c
index c1c6337..14f9da9 100644
--- a/drivers/gpu/drm/bochs/bochs_mm.c
+++ b/drivers/gpu/drm/bochs/bochs_mm.c
@@ -385,7 +385,7 @@ int bochs_gem_create(struct drm_device *dev, u32 size, bool iskernel,
 
  *obj = NULL;
 
- size = ALIGN(size, PAGE_SIZE);
+ size = PAGE_ALIGN(size);
  if (size == 0)
   return -EINVAL;
 
-- 
1.7.1